README.md

```diff
@@ -0,0 +1,68 @@
+[![image](https://travis-ci.org/aiidateam/aiida_core.svg?branch=develop)](https://travis-ci.org/aiidateam/aiida_core)
```

Comment:
could you please update the placeholders to something meaningful? It's already done in the link I provided.

Unfortunately, it happens quite frequently that one of the badges isn't loaded.